db/schema.rb

```diff
@@ -138,6 +139,7 @@
     t.uuid "subscription_id", null: false
     t.uuid "content_change_id", null: false
     t.index ["content_change_id"], name: "index_subscription_contents_on_content_change_id"
+    t.index ["created_at"], name: "index_subscription_contents_on_created_at"
```

Comment:
Do you know why this change is here? I can't see it in the migration?

Reply:
This isn't related to my change. I have a feeling that the schema was out of date. Or possibly someone manually edited the schema?

Reply:
Odd, there is an index on this field in production but not in staging. Since it's in production, seems like a good idea to have the `schema.rb` file match!

```
thomasleese@staging-email-alert-api-3:~$ govuk_app_console email-alert-api
irb(main):001:0> SubscriptionContent.connection.index_exists? :subscription_contents, :created_at
=> false
```

```
thomasleese@production-email-alert-api-3:~$ govuk_app_console email-alert-api
irb(main):001:0> SubscriptionContent.connection.index_exists? :subscription_contents, :created_at
=> true
```

Reply:
Interesting, thanks for looking into that!